### PR TITLE
Table uniq key

### LINKED ISSUE #55: Table - Alignement
Allow to change the text align by column to the left, right, ... or the comma for the number

### LINKED ISSUE #81: Table - Visible true by default
On each line, we need to add visible true.
Put the visible true by default to not have to add it for each line

### LINKED ISSUE #82: Table - Unique key
Each line need a unique key.

### PR DESCRIPTION
**Pull request by MarcoPOtard**
close #82 , close #81 , close #55

### CODE REVIEW

**Comment by MatouTechnologies** [comment]
This pull request has been deployed to Vercel.
<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://matou-technologies-purrfect-ui-storybook-92.vercel.app'>https://matou-technologies-purrfect-ui-storybook-92.vercel.app</a></td>
  </tr>
  <tr>
    <td><strong>🔍 Inspect:</strong></td>
    <td><a href='https://vercel.com/matou-technologies/purrfect-ui/4fbYAQwTyWTueT3xdwfChfWAm6vf'>https://vercel.com/matou-technologies/purrfect-ui/4fbYAQwTyWTueT3xdwfChfWAm6vf</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="deploy-preview-storybook"></hidden>

File: src/display/Table/Table.stories.tsx [hunk 1]
```diff
@@ -210,6 +108,38 @@ export const tableOnlyOneLineSelectable: Story = {
   ),
 };
 
+const dataRowsOneIsDisabled = [
```
Comment (Rcaradec) [inline src/display/Table/Table.stories.tsx:111]: dataRowsOneDisabled ?

File: src/display/Table/Table.stories.tsx [hunk 2]
```diff
@@ -255,6 +192,62 @@ export const tableEvenLinesBackgroundColored: Story = {
   ),
 };
 
+const dataRowsOneIsColored = [
```
Comment (Rcaradec) [inline src/display/Table/Table.stories.tsx:195]: dataRowsOneColored

File: src/display/Table/Table.stories.tsx (hunk 1 above)
Comment (MarcoPOtard) [reply src/display/Table/Table.stories.tsx]: Modifié

File: src/display/Table/Table.stories.tsx (hunk 2 above)
Comment (MarcoPOtard) [reply src/display/Table/Table.stories.tsx]: Modifié

**Review by Rcaradec — APPROVED** [review]

**Pull request merged**